### ISSUE TITLE
`Code -> Soft Wrap Long Lines` Does Not Affect New Windows, But `Tools -> Global Options -> Code -> Soft-wrap...` Does

### ISSUE THREAD

**Issue by emstruong** [rstudio/rstudio#14691]
### System details

    RStudio Edition : Desktop
    RStudio Version : RStudio 2023.12.1+402 "Ocean Storm" Release (4da58325ffcff29d157d9264087d4b1ab27f7204, 2024-01-29) for Ubuntu Jammy
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) rstudio/2023.12.1+402 Chrome/116.0.5845.190 Electron/26.2.4 Safari/537.36
    OS Version      : Ubuntu 22.04
    R Version       : 4.3.2

### Steps to reproduce the problem
- Have two R source code files open in the IDE
- Open one R source code file in a 'new window' using the pop-up button on the top left
- Select `Code -> Soft Wrap Long Lines`

### Describe the problem in detail
- It works for the R source code file in the main window, but not the new window
- You can 'fix' this issue by going to `Tools -> Global Options -> Code -> Soft-wrap...`, but then (presumably) all R source code files across all R Studio projects will be soft-wrapped.

### Describe the behavior you expected
- For soft-wrapping to be able to be applied to the new window

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@emstruong Thank you for raising this! I can also reproduce this in RStudio Desktop 2023.03.2+454 on Windows 11. But in 2024.04.0+735, I'm seeing the reverse, in which the when popping out the file in a new window results in the code always soft-wrapping, even if soft wrapping was off.

**Comment by emstruong** [comment]
No problem, @ronblum! For what it's worth, in my experience, when there are new windows, whether the option is 'actually' on or off seems to have a loose correspondence with the R Studio toolbar indicator of whether the option is checked-off or not. 

This issue goes beyond soft-wrapping and can also happen when one is trying to execute code with "Ctrl + Enter", where it'll execute code from the main window instead of the new window. But I don't have a reprex handy.

**Comment by ronblum** [comment]
@emstruong I think that what you're describing is in #11684? The fix for it is slated in the release that we're currently working on, Cranberry Hibiscus.

**Comment by emstruong** [comment]
> @emstruong I think that what you're describing is in #11684? The fix for it is slated in the release that we're currently working on, Cranberry Hibiscus.

Yes, that's it! Without knowing any of the R-Studio source code, I'd imagine that it's some common bug across both issue.

**Comment by ronblum** [comment]
@emstruong It might be caused by the same issue, but even so, I'll keep this often rather than merging issues, since we don't know for certain if a fix for one issue will fix both.
